core/enumerator.rbs

```diff
@@ -124,6 +124,43 @@
 class Enumerator[unchecked out Elem, out Return] < Object
   include Enumerable[Elem]
 
+  # <!--
```

Comment:
I copied this document from enumerator.c. Is that alright?
https://github.com/ruby/ruby/blob/master/enumerator.c#L3052-L3086

Reply:
I updated this comment by the result of CI (`rake annotate`?): https://github.com/ruby/rbs/actions/runs/6149970207/job/16686945264?pr=1516